LICENSE.mit

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2005-2014 Rich Felker, et al.
+Copyright (c) 2016 Harvey OS
```

Comment:
You've got copyright names here and in the files too. This is a bit unusual but it's up to you. Also in this file it's "Harvey OS", whatever that is -- is that a legal entity? If not, why is it here? But even then the name in the files is alvaro. Which is it?

Suggestion: here is what we did in coreboot: 
https://review.coreboot.org/cgit/coreboot.git/tree/

unmodified license. Then, in each file, we put the names with copyrights. That keeps it clear. What you're doing here is mixing up the issue of what the license is vs. who the contributors are and copyright dates.

The name LICENSE for the UCB release license is a questionable choice. It implies it covers the whole tree -- does it? Maybe 
LICENSE.plan9gpl
or something to make it clear it only applies to original Plan 9 sources? You've got lots of stuff in this tree not covered under LICENSE.